### PR TITLE
ADM-404 add JiraServiceTest

### PR DESCRIPTION
**Pull request by Eason-T**
## Summary
- Add jira board service test
...

## After

_Backend Coverage_

Screenshot 2023-03-22 at 16 06 49

## Note

_Null_

### CODE REVIEW

File: backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java [hunk 1]
```diff
@@ -25,7 +25,9 @@ public class JiraBoardConfigDTOFixture {
 
 	public static final String BOARD_ID = "123";
 
-	public static final String BOARD_NAME = "jira";
+	public static final String BOARD_NAME_JIRA = "jira";
```
Comment (guzhongren) [inline backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java:28]: Can we use existing variable in main code

**Comment by guzhongren** [comment]
Can u paste the code coverage screenshot here

File: backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java (hunk 1 above)
Comment (bhan-tw) [reply backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java:28]: Do you mean the name of BoardType?

```
JIRA("jira")
```

**Review by guzhongren — APPROVED** [review]

File: backend/src/main/java/heartbeat/service/board/jira/JiraService.java [hunk 2]
```diff
@@ -58,11 +58,6 @@ public class JiraService {
 	public static final List<String> FIELDS_IGNORE = List.of("summary", "description", "attachment", "duedate",
 			"issuelinks");
 
-	@PreDestroy
-	public void shutdownExecutor() {
-		taskExecutor.shutdown();
-	}
-
```
Comment (Teresasasa) [inline backend/src/main/java/heartbeat/service/board/jira/JiraService.java:65]: I am curious about why delete these codes. Don't we need to shutdown executor before all beans destroy?

**Pull request merged**